src/main/java/apoc/util/FileUtils.java

```diff
@@ -138,7 +138,20 @@ public static OutputStream getOutputStream(String fileName, OutputStream out) th
                 throw new RuntimeException(e);
             }
         } else {
-            outputStream = fileName.equals("-") ? out : new FileOutputStream(fileName);
+            if (fileName.equals("-")) {
+                outputStream = out;
+            } else {
+                boolean enabled = ApocConfiguration.isEnabled("import.file.use_neo4j_config");
+                if (enabled) {
+                    String importDir = ApocConfiguration.get("dbms.directories.import", "import");
```

Comment:
do we have constants for these? or move them into a method in apocconfig or fileutils

Reply:
ok, we put them into two methods into `FileUtils`